homeassistant/components/acmeda/base.py

```diff
@@ -72,11 +73,6 @@ def device_id(self) -> str:
         """Return the ID of this roller."""
         return self.roller.id
 
-    @property
-    def name(self) -> str | None:
-        """Return the name of roller."""
-        return self.roller.name
-
```

Comment:
Seems as `cover` is also based on this so needs some refactoring and can't just simply be removed e.g. leaving `cover` without a name.

Reply:
Set the _attr_name to None so it will default to the device name, which is self.roller.name